packages/opal-common/opal_common/sources/api_policy_source.py

```diff
@@ -76,39 +76,47 @@ async def get_initial_policy_state_from_remote(self):
         """init remote data to local repo."""
         async for attempt in AsyncRetrying(wait=wait_fixed(5)):
             with attempt:
-                await self.fetch_policy_bundle_from_api_source(
-                    self.remote_source_url, self.token
-                )
-                self.local_git = self.tar_to_git.create_local_git()
+                try:
+                    await self.fetch_policy_bundle_from_api_source(
+                        self.remote_source_url, self.token
+                    )
+                    self.local_git = self.tar_to_git.create_local_git()
+                except Exception:
```

Comment:
The create local git can also fail, should have a seperate exception an error log